Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/D7YIGPositionCalibration.py

```diff
@@ -53,6 +53,9 @@ def validateInputs(self):
         if self.getPropertyValue('FittingMethod') != 'None' and self.getProperty('CalibrationOutputFile').isDefault:
             issues['CalibrationOutputFile'] = 'The name of the calibration output needs to be defined'
 
+        if self.getPropertyValue('YIGPeaksFile') == '':
```

Comment:
Is this still needed?

Reply:
Not at all, I've just removed it from the last commit.